lib/keila_web/controllers/public_form_controller.ex

```diff
@@ -126,43 +126,96 @@ defmodule KeilaWeb.PublicFormController do
 
   @default_unsubscribe_form %Contacts.Form{settings: %Contacts.Form.Settings{}}
   @spec unsubscribe(Plug.Conn.t(), map()) :: Plug.Conn.t()
-  def unsubscribe(conn, %{
+  def unsubscribe(conn = %{method: "GET"}, %{
         "project_id" => project_id,
         "recipient_id" => recipient_id,
         "hmac" => hmac
       }) do
     if Mailings.valid_unsubscribe_hmac?(project_id, recipient_id, hmac) do
+      form = Contacts.get_project_forms(project_id) |> List.first() || @default_unsubscribe_form
+
+      conn
+      |> put_meta(:title, gettext("Unsubscribe"))
+      |> assign(:form, form)
+      |> assign(:project_id, project_id)
+      |> assign(:recipient_id, recipient_id)
+      |> assign(:hmac, hmac)
+      |> assign(:mode, :full)
+      |> assign(:unsubscribed, false)
```

Comment:
Instead of setting `@unsubscribed`, the successful unsubscription should simply render a different template.